src/angular/table/sort/sort-header.scss

```
@@ -60,6 +62,9 @@
   // Start off at 0 since the arrow may become visible while parent are animating.
   // This will be overwritten when the arrow animations kick in. See #11819.
   opacity: 0;
+  transition:
```

instead of duplicating $sbb-sort-animation-transition, you could use the other way:
```suggestion
    transition: {
      property: opacity, transform;
      duration: var(--sbb-animation-duration-6x);
      timing-function: var(--sbb-animation-easing); // TODO: Check if it looks ok.
  }
```